libmscore/figuredbass.cpp

```diff
@@ -17,10 +17,13 @@
 #include "system.h"
 #include "segment.h"
 #include "chord.h"
+#include "staff.h"
 #include "rest.h"
 #include "score.h"
 #include "sym.h"
 #include "xml.h"
+#include "utils.h"
+#include "mscore/scoreview.h"
```

Comment:
Can someone please tell me how to include code snippets in such a message as mgavioli and lasconic have done in this thread?  I know how to get the https link, but then how do I include it in the text here.  If I just insert the link, it appears as a clickable link, not showing the destination content.